z3c/dependencychecker/modules.py

```diff
@@ -308,7 +308,7 @@ def _parse_docstring(self, docstring):
                 try:
                     tree = ast.parse(code)
                 except SyntaxError:
```

Comment:
I wonder, should we log this at debug level? Not sure.

Reply:
Good question... I see that we have a couple of ``except SyntaxError``, so if we do, we should  do it in both.

While debugging myself I noticed this SyntaxError exceptions while working on the python 3 porting. There is a ``    >>> print 7`` on the ``sample1`` test project, which breaks on python 3, is that useful for users of z3c.dependencychecker? I'm not so sure.

As you are suggesting to add it at debug level, it is not going to be that noticeable anyway. I'm adding it. At least for us while debugging is going to be useful.